Quadriga/src/test/java/edu/asu/spring/quadriga/web/workbench/DeleteProjectControllerTest.java

```diff
@@ -0,0 +1,55 @@
+package edu.asu.spring.quadriga.web.workbench;
+
+import static org.junit.Assert.assertEquals;
+
+import java.security.Principal;
+import java.util.ArrayList;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.springframework.web.servlet.ModelAndView;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import edu.asu.spring.quadriga.exceptions.QuadrigaAccessException;
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+import edu.asu.spring.quadriga.service.workbench.IModifyProjectManager;
+
+public class DeleteProjectControllerTest {
+    
+    @InjectMocks
+    DeleteProjectController mockDeleteProjectController;
```

Comment:
should not be called mockDelete... just call it deleteProjectController or controllerUnderTest? Otherwise it sounds like you're mocking the controller.